retrofit2/src/test/java/com/linecorp/armeria/client/retrofit2/StreamingCallSubscriberTest.java

```diff
@@ -109,16 +130,18 @@ public void splitHeaders() throws Exception {
         subscriber.onSubscribe(subscription);
         subscriber.onNext(HttpHeaders.of(100));
         subscriber.onNext(HttpHeaders.of(200));
-        subscriber.onNext(HttpHeaders.of(HttpHeaderNames.CONTENT_TYPE, MediaType.PLAIN_TEXT_UTF_8.toString()));
-        subscriber.onNext(HttpData.ofUtf8("bar"));
+        subscriber.onNext(HttpHeaders.of(HttpHeaderNames.of("foo"), "bar")); // Trailers.
+        subscriber.onNext(HttpData.ofUtf8("baz")); // Ignored.
         subscriber.onComplete();
 
         verify(subscription, times(4)).request(1L);
 
         await().untilAsserted(() -> assertThat(callback.callbackCallingCount).isEqualTo(1));
-        await().untilAsserted(() -> assertThat(callback.response.header("content-type"))
-                .isEqualToIgnoringCase("text/plain; charset=utf-8"));
-        await().untilAsserted(() -> assertThat(callback.response.body().string()).isEqualTo("bar"));
+
+        TimeUnit.SECONDS.sleep(2);
```

Comment:
nit: Is this necessary?

Reply:
Just added because there's no way to retrieve trailers now. Will remove after we upgrade Retrofit and are able to get trailers which lets us know the end of stream.